src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -206,13 +206,13 @@ else if (DistributionType.LOCAL.equals(gradleProjectSettings.getDistributionType
     /**
      * Get the local wrapper path for Gradle that is in the project level
      *
-     * @param project liberty project
+     * @param wrapperDir location of gradle wrapper
      * @return the Graddle wrapper path to be executed or an exception to display
      * @throws LibertyException
      */
-    private static String getLocalGradleWrapperPath(Project project) throws LibertyException {
+    private static String getLocalGradleWrapperPath(String wrapperDir) throws LibertyException {
         String gradlew = SystemInfo.isWindows ? ".\\gradlew.bat" : "./gradlew";
-        File file = new File(project.getBasePath(), gradlew);
+        File file = new File(wrapperDir, gradlew);
```

Comment:
do you want to make the same change here as on the Maven side and change the name `file` to `wrapper`?

Reply:
actually, let's leave this for now. We can update the naming to be consistent with Maven another time.

Reply:
Good catch!